Localization/Translations/Generics.lua

```diff
@@ -442,22 +442,46 @@ local genericsLocales = {
         ["esMX"] = "Vendedores",
         ["enUS"] = true,
         ["zhCN"] = "供应商",
-        ["zhTW"] = false,
+        ["zhTW"] = "商人",
         ["esES"] = "Vendedores",
         ["frFR"] = "Vendeurs",
     },
-    ["Achievements"] = {
+	["Achievements"] = {
         ["ptBR"] = false,
         ["ruRU"] = "Достижения",
         ["deDE"] = "Erfolge",
         ["koKR"] = false,
         ["esMX"] = "Logros",
         ["enUS"] = true,
         ["zhCN"] = "成就",
-        ["zhTW"] = false,
+        ["zhTW"] = "成就",
         ["esES"] = "Logros",
         ["frFR"] = "Hauts faits",
     },
+	["Questie"] = {
+        ["ptBR"] = true,
+        ["ruRU"] = true,
+        ["deDE"] = true,
+        ["koKR"] = true,
+        ["esMX"] = true,
+        ["enUS"] = true,
+        ["zhCN"] = true,
+        ["zhTW"] = "任務-位置",
+        ["esES"] = true,
+        ["frFR"] = true,
+    },
+	["Questie Title"] = {
+        ["ptBR"] = "Questie",
+        ["ruRU"] = "Questie",
+        ["deDE"] = "Questie",
+        ["koKR"] = "Questie",
+        ["esMX"] = "Questie",
+        ["enUS"] = "Questie",
+        ["zhCN"] = "Questie",
+        ["zhTW"] = "任務位置提示",
+        ["esES"] = "Questie",
+        ["frFR"] = "Questie",
+    }
```

Comment:
Can you explain what this is about?

Reply:
Can you help me with this @class2u ? 😃

Reply:
Do you think it is a good idea to translate these usages and keep the code you added? 🤔